src/components/indicators/IndicatorCard.tsx

```diff
@@ -200,6 +209,7 @@ interface IndicatorCardProps {
     unit: string;
   } | null;
   disabled?: boolean | null;
+  customHref?: string | null;
```

Comment:
Nitpick: We only need to type for `null` _and_ `undefined` when the value comes from the backend and could be either. Since we create this type we can settle on one or the other.

We use `null` to represent the intentional absence of a value and  `undefined` when a variable hasn't been assigned a value. 

```suggestion
  customHref: string | null;
```

Reply:
Also the default parameter `= null` isn't necessary since the value is always either a string or null.